docs/explanation/gno-tooling/cli/gnofaucet.md

```diff
@@ -0,0 +1,53 @@
+---
+id: gno-tooling-gnofaucet
+---
+
+# gnofaucet
+
+`gnofaucet` is a server for distributing GNOT, the gas currency of Gnoland, to specific addresses in a local chain. Interact with the `gnofaucet` from an address with an empty balance in your locally built testnet to fuel it with GNOT to pay for transactions.
+
+## Run `gnofaucet` Commands
+
+Enable the faucet using the following command.
+
+```bash
+gnofaucet serve
+```
+
+#### **Options**
+
+| Name                      | Type    | Description                                                                          |
+| ------------------------- | ------- | ------------------------------------------------------------------------------------ |
+| `chain-id`                | String  | The id of the chain (required).                                                      |
+| `gas-wanted`              | Int64   | The maximum amount of gas to use for the transaction (default: `50000`)              |
+| `gas-fee`                 | String  | The gas fee to pay for the transaction.                                              |
+| `memo`                    | String  | Any descriptive text (default: `""`)                                                 |
+| `test-to`                 | String  | Test address (optional) 부연설명 필요                                                |
```

Comment:
> 부연설명 필요

Is this in the code?

Reply:
looks like typo from our dev portal, we can get rid of it

Reply:
Fixed:

[6bf83af](https://github.com/gnolang/gno/pull/1046/commits/6bf83af594267040a008261985c5ab82db6145d3)